Prevent re-render of preview overlay when there are no changes (#3096)
vivid-planet/comet: .changeset/metal-waves-add.md

---
"@comet/cms-site": patch
---

> Prevent the admin's block-preview from becoming unresponsive when rendering an `input` in the frontend

Reviewer: I'd either use a generic message

```suggestion
Prevent infinite recalculation of preview overlays in the block preview
```

or remove the term "frontend", since both Admin and site are technically frontends, which can lead to confusion.

```suggestion
Prevent the block-preview from becoming unresponsive when rendering an `input`
```

Author: Good point, I like the second one
